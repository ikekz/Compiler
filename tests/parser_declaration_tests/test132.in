function f(a : integer) : array of array [1..2] of double;
begin
end;
begin
end.