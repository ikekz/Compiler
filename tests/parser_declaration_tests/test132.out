integer        type           integer
double         type           double
char           type           char
f              function       return type: 
                              array of array [1..2] of double
                              args:
                              a              valueparam     integer
                              locals:
                              none
